general_rules/PenaltiesBonuses.tex

```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
 	\item It is the decision of the \iaterm{Technical Committee}{TC} if (and to which degree) the bonus score is granted.
 \end{enumerate}
 
+\subsection{Bonus for perceived social intelligence}\label{rule:perceived_intelligence}
+\begin{enumerate}	
+    \item For the test \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
+    
+    \item This bonus, ranging from 0 to 50, depends on the robot's social performance which will be assessed by Referees using a specially designed scale in a survey.
+
+    \item Every team seeking a social assessment should inform the referee before the test. that they want to be evaluated using perceived social intelligence.    
+  
+    \item After the test is completed, the evaluators will fill out the form accessible via the QR code in Figure \ref{fig:qr-survey}, or through this \href{https://forms.office.com/Pages/ResponsePage.aspx?id=6sSEXw03nkuDDHVvi_G1H7VNGCdGFtZJs0ryJVVWtCFUQVFSWDlYM0FHRVA2QllIT0tOQjI2QUcxQi4u}{link}.
+    
+    \item The referee has the authority to skip the social assessment test if they believe the robot's performance is not suitable for measurement.
```

Comment:
But is that fair? If the team wants to be evaluated, they should be allowed to get it. The referee can suggest it, but I think the team should be allowed to decide.